step-templates/azure-linux-install-octopus-tentacle.json

```diff
@@ -0,0 +1,145 @@
+      "Label": "NSG Name",
+      "HelpText": "The name of the azure network security group to create ",
+      "DefaultValue": "",
+      "DisplaySettings": {
+        "Octopus.ControlType": "SingleLineText"
+      }
+    },
+    {
+      "Id": "36c6d441-9fe8-4ea2-957a-2135aa9d1687",
+      "Name": "installLinuxTentacle.azNsgRulePriority",
+      "Label": "Azure NSG Rule Priority",
+      "HelpText": "the priority to assign to the NSG rule created for the octopus tentacle. Defaults to 400",
+      "DefaultValue": "400",
+      "DisplaySettings": {
+        "Octopus.ControlType": "SingleLineText"
+      }
+    },
+    {
+      "Id": "e4358aec-2798-412a-bcd2-1c7e41a0728d",
+      "Name": "installLinuxTentacle.azVmName",
+      "Label": "VM Name",
+      "HelpText": "The name of the virtual machine to target when ",
+      "DefaultValue": "",
+      "DisplaySettings": {
+        "Octopus.ControlType": "SingleLineText"
+      }
+    },
+    {
+      "Id": "8dbe0a29-4e0f-43f8-8ef2-ae23a2884e85",
+      "Name": "installLinuxTentacle.octoServerThumb",
+      "Label": "Server Thumbprint",
+      "HelpText": "The Thumbprint of the octopus server",
+      "DefaultValue": "",
+      "DisplaySettings": {
+        "Octopus.ControlType": "Sensitive"
+      }
+    },
+    {
+      "Id": "a0859551-6c84-45d1-af2b-c57becb4e8ef",
+      "Name": "installLinuxTentacle.octoApiKey",
+      "Label": "API Key",
+      "HelpText": "The API key used to configure the tentacle.",
+      "DefaultValue": "",
+      "DisplaySettings": {
+        "Octopus.ControlType": "Sensitive"
+      }
+    },
+    {
+      "Id": "dd07815d-bfb7-43b9-90f8-21ec5a2d2953",
+      "Name": "installLinuxTentacle.octopusRoles",
+      "Label": "Roles",
+      "HelpText": "Roles to assign to this tentacle installation. <br />\n*Note: Each role should be on it's own line*",
+      "DefaultValue": "",
+      "DisplaySettings": {
+        "Octopus.ControlType": "MultiLineText"
+      }
+    },
+    {
+      "Id": "d3ffb38e-7fcf-4547-be9a-6cd8e46f8e5e",
+      "Name": "installLinuxTentacle.octopusEnvironments",
+      "Label": "Environments",
+      "HelpText": "Environments to assign this tentacle installation to.<br />\n*Note: Each environment should be on its own line*",
+      "DefaultValue": "",
+      "DisplaySettings": {
+        "Octopus.ControlType": "MultiLineText"
+      }
+    },
+    {
+      "Id": "15150827-7bd3-4146-a798-66344851f602",
+      "Name": "instrallTentacle.octoServerUrl",
+      "Label": "Server Url",
+      "HelpText": "The server url to register the tentacle with.  Defaults to the base url",
+      "DefaultValue": "#{Octopus.Web.BaseUrl}",
+      "DisplaySettings": {
+        "Octopus.ControlType": "SingleLineText"
+      }
+    },
+    {
+      "Id": "9bcd6bb5-2db3-453f-b4f6-6a78f3c39b59",
+      "Name": "installLinuxTentacle.tentacleType",
+      "Label": "Tentacle Type",
+      "HelpText": "Select between a listing or polling tentacle",
```

Comment:
Real minor typo: `listing` instead of `listening`

Reply:
addressed in changeset f685690